study.md

```diff
@@ -49,15 +49,16 @@ should contain two strings, "drinking beer" and "conspiracy theories".  The keys
 should be symbols.
 
 ```ruby
-#your answer here
+dale[:middle_name].push('Alvin')
```

Comment:
`dale[:middle_name]` is a `String` and trying to invoke `push` on it will cause an error.